.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -27,6 +33,10 @@ jobs:
   eager-package-main:
     uses: ./.github/workflows/eager-package-main.yaml
     secrets: inherit
+  build-docs:
+    needs: fd-unit-tests
```

Comment:
See if you can make it depend on both sets of tests `sd-unit-tests` and `fd-unit-tests`